src/main/java/info/movito/themoviedbapi/TmdbTrending.java

```diff
@@ -32,8 +32,23 @@ public class TmdbTrending extends AbstractTmdbApi {
      * @throws TmdbException If there was an error making the request or mapping the response.
      */
     public MultiResultsPage getAll(TimeWindow timeWindow, String language) throws TmdbException {
+        return getAll(timeWindow, language, 1);
+    }
+
+    /**
+     * <p>Get the trending movies, TV shows and people.</p>
+     * <p>See the <a href="https://developer.themoviedb.org/reference/trending-all">documentation</a> for more info.</p>
+     *
+     * @param timeWindow The time window for the trending media.
+     * @param language   nullable - The language to query the results in. Default: en-US.
+     * @param page       nullable - The page of results to return. Default: 1.
+     * @return The trending media.
+     * @throws TmdbException If there was an error making the request or mapping the response.
+     */
+    public MultiResultsPage getAll(TimeWindow timeWindow, String language, Integer page) throws TmdbException {
         ApiUrl apiUrl = new ApiUrl(TMDB_METHOD_TRENDING, "all", timeWindow.getValue());
         apiUrl.addLanguage(language);
+        apiUrl.addPage(page == null ? 1 : page);
```

Comment:
You don't need to check for null when calling `apiUrl.addPage`; `info.movito.themoviedbapi.tools.ApiUrl#addPage` checks for null.

Reply:
Fixed!